webui-src/app/mail/mail_resolver.js

```diff
@@ -34,11 +35,27 @@ const sections = {
   drafts: require('mail/mail_draftbox'),
   sent: require('mail/mail_sentbox'),
 };
-
+const tagselect = {
+  showval: 'Tags',
+  opts: ['Tags','Important', 'Work', 'Personal']
+}
 const Layout = {
   oninit: Messages.load,
   view: (vnode) =>
+
     m('.tab-page', [
+      m("button[id=composebtn]", { onclick: () => { widget.popupMessage(m(compose)); } }, "Compose"),
+      m('select[id=tags]', {
+        value: tagselect.showval,
+        onchange: e => {
+          tagselect.showval = tagselect.opts[e.target.selectedIndex]
+        }
+      }, [
+        tagselect.opts.map(o => m('option', { value: o }, o.toLocaleString()))
+      ]),
+      m(util.SearchBar, {
+        list: Object.assign({}, {}, {}),
```

Comment:
What is the reason for this? Why not simply do:
```suggestion
        list: {},
```

Reply:
Right I will make the change for now.
I was actually trying to implement search.

Reply:
I will run eslint and then push these changes.